specification/security/resource-manager/readme.md

```diff
@@ -81,6 +85,7 @@ These settings apply only when `--tag=package-preview-2021-08` is specified on t
 input-file:
   - Microsoft.Security/preview/2021-08-01-preview/standards.json
   - Microsoft.Security/preview/2021-08-01-preview/assignments.json
+  - Microsoft.Security/preview/2021-12-01-preview/scanners.json
```

Comment:
>2021-12-01-preview [](http://example.com/codeflow?start=31&length=18)

Can you introduce a new tag for the newer api version? Generally newer api versions are not included in the older api version tags like package-preview-2021-08